02-electric/electric-section-one/electric-section-one.css

```diff
@@ -1,45 +1,114 @@
+.electric-section-one-container {
+  padding: 15px;
+}
 
-.electric-section-one-container{
-padding: 15px;
+.electric-section-one-container > * {
+  border: solid 1px black;
 }
 
-.electric-section-one-container >*{
-    border: solid  1px black;
+.electric-section-one-subcontainer {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-content: center;
+  row-gap: 15px;
+  padding: 25px;
+}
+.electric-section-one-subcontainer > * {
+  border: solid 1px black;
 }
 
-.electric-section-one-subcontainer{
-    display: grid;
-    grid-template-columns: 1fr;
-    justify-content: center;
-    row-gap: 15px;
-    padding: 25px
-  
-  }
-  .electric-section-one-subcontainer > *{
-      border: solid  1px black;
-  }
-  
-  .electric-section-one-img-container{
+/*.electric-section-one-img-container{
       grid-column: 1/2;
       
   }
   .electric-section-one-img{
     display: block;
     width: 100%;
+}*/
+
+.electric-section-one-title {
+  grid-column: 1/2;
+  text-align: center;
 }
+
+.electric-section-one-subcontainer {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-content: center;
+  row-gap: 15px;
+  padding: 25px;
+}
+
+/*****PART ONE******/
+.electric-section-one-part-one-img-container {
+  grid-column: 1/2;
+}
+.electric-section-one-part-one-img {
+  display: block;
+  width: 100%;
+}
+
+.electric-section-one-part-one-title {
+  grid-column: 1/2;
+  text-align: center;
+  font-size: var(--title-size-mobile);
+}
+
+.electric-section-one-part-one-content {
+  grid-column: 1/2;
+  text-align: justify;
+}
+
+/*****PART two******/
+.electric-section-one-part-two-img-container {
+  grid-column: 1/2;
+}
+.electric-section-one-part-two-img {
+  display: block;
+  width: 100%;
+}
+
+.electric-section-one-part-two-title {
+  grid-column: 1/2;
+  text-align: center;
+  font-size: var(--title-size-mobile);
+}
+
+.electric-section-one-part-two-content {
+  grid-column: 1/2;
+  text-align: justify;
+}
+
+/*****PART two******/
+.electric-section-one-part-three-img-container {
+  grid-column: 1/2;
+}
+.electric-section-one-part-three-img {
+  display: block;
+  width: 100%;
+}
+
+.electric-section-one-part-three-title {
+  grid-column: 1/2;
+  text-align: center;
+  font-size: var(--title-size-mobile);
   
-  .electric-section-one-title{
-      grid-column: 1/2;
-     text-align: center;
-  }
-  .electric-section-one-content{
-      grid-column: 1/2;
-     text-align: justify;
-  }
-  
-  .electric-section-one-btn{
-    grid-column: 1/2;
-    width:90px;
-    height: 40px ;
-    justify-self: center;
-  }
\ No newline at end of file
+}
+
+.electric-section-one-part-three-subtitle {
+  grid-column: 1/2;
+  text-align: center;
+  font-size: var(--subtitle-size-mobile);
+}
+
+.electric-section-one-part-three-content {
+  grid-column: 1/2;
+  text-align: justify;
+}
+
+.electric-section-one-btn {
+  grid-column: 1/2;
```

Comment:
in a 1 column grid ithe cell automatically spans the full width of a grid
so we don't need this.
(except on desktop with media queries we might need to put it back for the different transitions between media queries... but let's remove it for now)

Reply:
done